<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="django.contrib.auth.tests.forms.AuthenticationFormTest" tests="3" time="0.002">
	<testcase classname="django.contrib.auth.tests.forms.AuthenticationFormTest" name="test_inactive_user" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.forms.AuthenticationFormTest" name="test_invalid_username" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.forms.AuthenticationFormTest" name="test_success" time="0.001"/>
	<system-out/>
	<system-err/>
</testsuite>
